build/rbac-setup.sh

```diff
@@ -11,7 +11,7 @@
 # DIR="$( find $HOME -type d -name "search-e2e-test" )"
 # using abosolute path, relative path is not working in travis
 
-set -e
+# set -e
```

Comment:
I'm ok with disabling this while we integrate with prow, but let's add an `echo` to log a message to remind us to revert this change.
```suggestion
# set -e
echo "WORKAROUND: Temporarily allowing `rbac-setup.sh` to continue with errors while we setup the prow build."
```

Reply:
Re-added this line. This should be fixed.